### PR TITLE
Docker with parallel lint

### PR DESCRIPTION
**Pull request by tobiasKaminsky**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -29,21 +29,22 @@ pipeline:
...
+        FLAVOUR: Lint
 
 matrix:
   FLAVOUR:
     - Generic
     - Gplay
     - Modified
+    - Lint
 
 branches: master
```
Comment (mario) [inline .drone.yml]: newline

File: .drone.yml [hunk 2]
```diff
@@ -29,21 +29,22 @@ pipeline:
...
+    when:
+      matrix:
+        FLAVOUR: Lint
 
 matrix:
   FLAVOUR:
     - Generic
     - Gplay
     - Modified
+    - Lint
```
Comment (mario) [inline .drone.yml]: Uhm? Lint flavour?

Comment (tobiasKaminsky) [inline .drone.yml]: Yeah, the purpose of this is to have lint executed in parallel.

**Comment by AndyScherzinger** [comment]
👍

**Comment by mario** [comment]
👍

**Pull request merged**